packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -42,54 +49,44 @@ const TimeDuration = styled.time`
   margin: 0 ${GEL_SPACING_HLF};
 `;
 
-const IndexAlsosMediaIndicator = styled.span`
```

Comment:
We will have to also update the `IndexAlsos` straight after to use the `MediaIndicatorWrapper` with the `isLine` prop instead since this `IndexAlsosMediaIndicator` won't exist anymore.

Reply:
Created [Psammead](https://github.com/bbc/psammead/issues/3039) and [Simorgh](https://github.com/bbc/simorgh/issues/5340) issues 👍